pyqmix_backend/run.py

```diff
@@ -1,5 +1,5 @@
 import webbrowser
-from .backend_app import app
+from pyqmix_backend.backend_app import app
```

Comment:
Was this a required change? I thought the previous relative import was nice… But I have no hard feelings here ;)

Reply:
It wasn't just a style change. The executable would crash and in the end I traced it back to this line. I suspect the relative import only worked previously because I specified pyqmix_backend as a source root in my PyCharm settings. Since we don't specify the source root settings in our documentation I really think this commit has the better approach.

Reply:
Wow, good job tracking this down! Sounds like an awful lot of work!